assets/prototype/application/services/apollo/status/useStatusManager.ts

```diff
@@ -33,7 +33,7 @@ const useStatusManager = (): StatusManager => {
 	 * @param {string}  typeName  The plural type name like "datetimes", "tickets", "prices"
 	 * @param {boolean} value Value of the flag
 	 */
-	const setIsLoading: StatusSetter = (typeName: TypeName, value: boolean = true): void => {
```

Comment:
Fix for:
![14_jan_4](https://user-images.githubusercontent.com/1477453/72342884-39de6980-36d6-11ea-98f3-f8a725746465.png)